docs/HOWTO.md

```diff
@@ -1,17 +1,36 @@
+# How-To at a glance
+
+<div align="right" markdown="1">
+
 *[Read this in other languages](README.md#translations)*
 
-Welcome to Free-Programming-Books! We welcome new contributors; even those making their very first pull request on Github. If you're one of those, here are some resources that might help:
+</div>
+
+**Welcome to `Free-Programming-Books`!**
+
+We welcome new contributors; even those making their very first Pull Request (PR) on GitHub. If you're one of those, here are some resources that might help:
+
+* [About pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+* [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)
+* [YouTube - GitHub Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [YouTube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [YouTube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Don't hesitate to ask questions; every contributor started with a first PR. So... why not join our [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community.
+
+<details align="center" markdown="1">
+<summary>Click to see users vs. time graphs.</summary>
+
+[![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)
 
-* [About Pull Requests](https://help.github.com/articles/about-pull-requests/)
-* [Creating a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
-* [Github Hello World](https://guides.github.com/activities/hello-world/)
-* [Youtube - Github Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
-* [Youtube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
-* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)
 
+NOTE: Contribution spikes use to match with the [Hacktoberfest event](https://hacktoberfest.digitalocean.com) dates.
```

Comment:
> If you like, I can write something.

Docs improvements are always welcome, I tought, since helps to people understand the state and proupose of the repo. Alfter remove hacktoberfest mention, I rather put it at any README.md joint with a badge like 

![Hacktoberfest 2021](https://img.shields.io/github/hacktoberfest/2021/EbookFoundation/free-programming-books)

that needs be updated yearly. Then, no translation are needed